src/reconcile/rec_col.c

```diff
@@ -1418,10 +1431,12 @@ __wt_rec_col_var(
                     data = cbt->iface.value.data;
                     size = (uint32_t)cbt->iface.value.size;
                     update_no_copy = false;
+                    dictionary = true;
```

Comment:
Why in this case dictionary is in use? Should it be
dictionary = btree->dictionary?

Reply:
I think it's a good idea to replace `dictionary = true` with `dictionary = btree->dictionary`. It will simplify the condition later in `__rec_col_var_helper`.

We are setting it to `true` as we are applying an update and we try to use the dictionary as often as possible, every time a _new_ value comes in.